src/mod.ts

```diff
@@ -98,6 +98,15 @@ import type {
 import type {
   MiniProgramPayload,
 }                             from './schemas/mini-program.js'
+import type {
+  VideoPostPayload,
+}                             from './schemas/video-post.js'
+import type {
+  CommentPayload,
```

Comment:
How about we reuse `Message` for comments?

Reply:
Do you mean discard `Comment` class ? Or only reuse `MessagePayload` for `CommentPayload`?

Reply:
If so, how do you think about it?

Reply:
Agree, https://github.com/wechaty/wechaty/issues/2245#issuecomment-916674263